lra/index.html.haml

```diff
@@ -0,0 +1,42 @@
+---
+layout: rightcol
+rightcol_partial: rightcol-documentation.html.haml
+title: Long Running Actions (LRA)
+documentation_highlight: true
+---
+
+.row
+  .col-md-12
+    %h1 Long Running Actions (LRA)
+    %p
+      Long Running Actions (abbreviated as
+      %code
+        LRA
+      ) is a MicroProfile specification that introduces APIs for services to coordinate activities.
+      For more information check site
+      %a{:href => "https://github.com/eclipse/microprofile-lra"} https://github.com/eclipse/microprofile-lra
+      \.
+    %p
+      Narayana implements the LRA specification.
+
+
+.row
+  .col-md-12
+    %h2
+      Certifications
+    %ul
+      %li
+        %a{:href => relative("/certifications/microprofile-lra-1.0/index.html")} LRA 1.0 Compatible Certification Request (CCR)
+
+.row
+  .col-md-12
+    %h2 Narayana LRA Coordinator OpenAPI Schema
+    %ul.list-group
+      %li.list-group-item
+        %a{:href => relative("/lra/api/1.0/openapi.yaml")}
```

Comment:
@ochaloup I think because it is relative this should be something like /api rather than /lra (I will add the same comment below)

Reply:
@tomjenkinson I don't know how the `awesturct` build and the haml works exactly but this way the links *works* nicely. I think relative is to the root of the site - for example how the `relative` is used in the rightcol navigation which is used in all subpages of Documentation - https://github.com/jbosstm/narayana.io/blob/develop/_partials/rightcol-documentation.html.haml

Reply:
I thought I tried this yesterday from the build output and it didn't seem to work for me. I will try again

Reply:
It's not working for me @ochaloup - from a download of the build output if I am on _site/lra/index.html and click on "Version: 1.0, format: YAML" it get's a 404 but I think it might not be the location, I don't think I can find the file in the _site

Reply:
I have changed the list of files that are archived and am re-running

Reply:
That was it - I will resolve this conversation and the other related one